mkdocs/themes/mkdocs/js/base.js

```diff
@@ -26,6 +27,37 @@ $(document).ready(function() {
         $search_modal.find('#mkdocs-search-query').focus();
     });
 
+    // Keyboard navigation
+    document.addEventListener("keydown", function(e) {
+        if ($(e.target).is(':input')) return true;
+        var key = e.which || e.keyCode || window.event && window.event.keyCode;
+        var page;
+        switch (key) {
+            case 39:   // right arrow
+                page = $('[role="navigation"] a:contains(Next):first').prop('href');
+                break;
+            case 37:   // left arrow
+                page = $('[role="navigation"] a:contains(Previous):first').prop('href');
+                break;
+            case 83:   // s
+                if (e.altKey) {
```

Comment:
Curious why you saw the need to require the `Alt` key here? I'm fine with it, just wondering why we can't use an `s` alone.